templates/vue.webapp/src/components/Tech.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
+    <ul v-for="technology in technologies" v-bind:key="technology.name" class="technologies">
+      <li>
+        <b>{{technology.name}}</b>: {{technology.details}}
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  name: 'Tech',
+  data() {
+    return {
+      technologies: []
+    };
+  },
+  mounted() {
+    console.log(process.env.ROOT_API)
```

Comment:
I would suggest to remote this console log

Reply:
Sure. Thank you! I forgot about this line